CommonBase/MissionScripts/Arsenal/arsenalGeneral.sqf

```diff
@@ -1331,57 +1327,25 @@ private _miscItems = [
     "ACRE_VHF30108"
 ];
 
-{
-    if (!isNull(_x)) then {
-
-        clearWeaponCargoGlobal _x;
-        clearMagazineCargoGlobal _x;
-        clearBackpackCargoGlobal _x;
-        clearItemCargoGlobal _x;
-
-        [_x, false, false] call ace_arsenal_fnc_initBox;
```

Comment:
`[_x, false, false] call ace_arsenal_fnc_initBox;`

This needed to actually create the arsenal, the addvirtualitems won't actually make it appear